sym/manifest.json

```diff
@@ -0,0 +1,79 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "d81d1dd3-d5e8-4373-98a6-f95b797b3f9d",
+    "app_id": "sym",
+    "display_on_public_website": true,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Send Sym Audit Logs to Datadog",
+        "title": "Sym",
+        "media": [
+            {
+                "media_type": "video",
+                "caption": "A Sym Overview Video",
+                "image_url": "images/sym_video_thumbnail.jpg",
+                "vimeo_id": 846654213
+            },
+            {
+                "media_type": "image",
+                "caption": "Sym helps you build access and approval workflows using your native platform engineering tools",
+                "image_url": "images/home_hero_image.png"
+            },
+            {
+                "media_type": "image",
+                "caption": "Define and deploy access rules in Terraform",
+                "image_url": "images/define_deploy.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "Request, approve, and deny anywhere in Slack",
+                "image_url": "images/request_approve.jpg"
+            },
+            {
+                "media_type": "image",
+                "caption": "An example of the Sym Overview Dashboard",
```

Comment:
Note that images need to have at least a 16:9 aspect ratio or else they will be cut off. You can learn more in our [docs on media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).